cmd/pdfgrab/main.go

```
@@ -66,7 +75,7 @@ func run(args []string, stdout, stderr io.Writer) error {
 		printUsage(stdout)
 		return nil
 	case "version", "-v", "--version":
-		fmt.Fprintln(stdout, "pdftable v0.3.0")
+		fmt.Fprintln(stdout, "pdfgrab "+version)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)